packages/vuetify/src/util/helpers.ts

```diff
@@ -365,6 +365,17 @@ export function debounce (fn: Function, delay: number) {
   }
 }
 
+export function throttle<T extends (...args: any[]) => any> (fn: T, limit: number) {
```

Comment:
This type of functionality was removed last May https://github.com/vuetifyjs/vuetify/commit/5aa23f15bf20127256e64561826bcfb2c0254006

Reply:
Does that mean you prefer to not have throttle helper function at all or that you would like to change the implementation of it to be like the one the was removed?

Reply:
I just want to understand why it was removed. Was it because after removing its use within the `v-color-picker` it had no purpose? Or was it because the throttle caused issues. I'm leaning towards the former, because this seems like it would be useful.

I'm not sure if @KaelWD can chime in but if not, I'd be interested in exploring other areas where this could be used.